js/imgur.js

```diff
@@ -61,6 +102,7 @@ weechat.factory('imgur', ['$rootScope', function($rootScope) {
             // Remove progress bar
             currentProgressBar.parentNode.removeChild(currentProgressBar);
 
+            console.log(xhttp.status);
```

Comment:
this should probably be removed before we merge

Reply:
Corrected.